packages/wallet/src/redux/sagas/multiple-action-dispatcher.ts

```diff
@@ -0,0 +1,9 @@
+import { MultipleWalletActions } from '../actions';
+import { take, put } from 'redux-saga/effects';
+
+export function* multipleActionDispatcher() {
+  while (true) {
+    const multipleWalletActions: MultipleWalletActions = yield take('WALLET.MULTIPLE_ACTIONS');
+    yield multipleWalletActions.actions.map(action => put(action));
```

Comment:
Does this work? Should `yield` apply to the `map` function or the `put` function?

Reply:
It works: first thing I tries was to `yield put` inside the loop, but that *doesn't* work.

Reply:
I think I understand what is going now. The saga is a generator that `yield`s an array. Then, the iterator (in this case the saga framework), processes the contents of the array.